### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by Yavin**
ignore me

### CODE REVIEW

**Comment by Yavin** [comment]
test comment
[google website](http://google.com)
alt

File: index.html [hunk 1]
```diff
@@ -53,8 +53,9 @@
 <img src="forkit.gif" id="octocat" alt="" />
 
 <p>
-  Fork me? No, fork you!
+  Fork me? No, fork you! 2
 </p>
+aaa
```
Comment (Yavin) [inline index.html:58]: some stupid message

Comment (Yavin) [inline index.html:58]: will that comment be merged?

File: index.html [hunk 2]
```diff
@@ -53,8 +53,9 @@
 <img src="forkit.gif" id="octocat" alt="" />
 
 <p>
-  Fork me? No, fork you!
```
Comment (Yavin) [inline index.html:56]: comnet jus above

**Comment by Yavin** [comment]
succesly merge 2 commits into 1, hooray!

**Comment by bubblzo** [comment]
hi

**Pull request closed without merging**